client/app/house-info/component/LeaveHouse.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react'
+import { FiLogOut } from 'react-icons/fi'
+import { Button } from '@/_components/ui'
+import { HouseType } from '@/_types'
+import { AdminWarningModal } from '@/account/components'
+import { useLeaveHouse } from '../hooks'
+import { LeaveHouseModal } from '.'
+
+type LeaveHouseProps = {
+  userHouse: HouseType
+  userId: number
+}
+
+const LeaveHouse = ({ userHouse, userId }: LeaveHouseProps) => {
+  const [isOpenLeaveHouseModal, setIsOpenLeaveHouseModal] = useState(false)
+  const [isOpenAdminWarningModal, setIsOpenAdminWarningModal] = useState(false)
+  const { onRemove, isPending } = useLeaveHouse()
+
+  const handleLeaveHouse = () => {
+    const isAdmin = userHouse.houseMembers.filter((member) => member.isAdmin)
```

Comment:
`adminMembers` instead `isAdmin` makes more sense!

Reply:
Fixed it.
c5544a21eb4c19135a287c72b2ffcaf02d117bb3